mobile/examples/Maui/MauiVisionSample/MauiVisionSample/MauiVisionSample.csproj

```
@@ -55,8 +57,12 @@
 	</ItemGroup>
 
 	<ItemGroup>
-	  <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.12.1" />
-	  <PackageReference Include="SkiaSharp" Version="2.88.6" />
+		<PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.18.0" />
+		<PackageReference Include="SkiaSharp" Version="2.88.7" />
+		<PackageReference Include="Microsoft.Maui.Controls" Version="8.0.14" />
+		<PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.14" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="8.0.0" />
+		<PackageReference Include="Microsoft.ML.OnnxRuntime.Extensions.Dummy" Version="0.10.0" />
```

```suggestion
		<PackageReference Include="Microsoft.ML.OnnxRuntime.Extensions.Dummy" Version="0.12.0" />
```

The xcframework for maccatalyst is more correct in 0.12.0. Apple started enforcing that it needed the 'Current' symlink etc.  on some platforms.